src/adapter.ts

```diff
@@ -70,6 +71,8 @@ export const mikroOrmAdapter = (
             Reflect.deleteProperty(input, "id")
           }
 
+          ensureReferencesAreLoaded(metadata, input)
```

Comment:
I believe `adapter.create` is not the only place where we need to get references to ensure Mikro ORM will not create entities where it shouldn't. The `adapter.update` is affected to, I think. Ideally this needs a test case.